src/docs/product/projects/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Projects
+sidebar_order: 1
+description: "View all the projects you're a member of, then dive into their details quickly."
+---
+A _Project_ connects the error and transaction events in your application to the team members assigned to that project.
+
+You can create a project for a particular language or framework, and typically do so when you first start using Sentry, though you can add projects as you need. For example, you might have separate projects for your API server and frontend client. Setting up multiple projects to reflect your application landscape with finer granularity will go a long way in helping with visibility and triage.
```

Comment:
To go with the above, I would slightly change language: You create a project _for_ a service and _select_ the platform / language.